package.json

```diff
@@ -8,7 +8,8 @@
   "workspaces": [
     "langchain",
     "examples",
-    "docs"
+    "docs",
+    "api-docs"
```

Comment:
Would prefer not to have a new workspace if possible - can we move into `docs` and add new commands there?